bench/wtperf/wtperf.c

```diff
@@ -1845,8 +1847,15 @@ execute_workload(WTPERF *wtperf)
         last_backup = wtperf->backup_ops;
     }
 
-/* Notify the worker threads they are done. */
 err:
+    /*
+     * If the start timer is uninitialized, the test didn't run. Set the test time anyway to avoid
+     * dividing by zero in reporting.
+     */
+    wtperf->testsec =
+      start_usec == 0 ? 1 : (uint32_t)((testutil_time_us(NULL) - start_usec) / WT_MILLION);
+
```

Comment:
If there's some situation where the test stops quickly and less than 1 second has elapsed, we'll still get divide by zero.  So I think we need an extra check (with warning message?)

Reply:
Good point! 

Since we're measuring time in microseconds, it seems that we could still report useful/accurate numbers on a short-lived test run. I'll see what I can do to improve this.

Reply:
I found a simpler approach. The main execution thread sits in a loop sleeping for one second, then checking whether any of the end conditions have been met. So we can just count the iterations of this loop to determine how long the test ran.

It's a bit less accurate than using the system clock. But it's what we were already doing with the `run_time` parameter. So now we just measure time the same way with the `run_ops` parameter.

@ddanderson tagging you for re-review in case you have further thoughts here.